shoulder_pan_joint: 0.227
shoulder_lift_joint: -0.9948
elbow_joint: 1.1285
wrist_1_joint: -2.526
wrist_2_joint: -1.551
wrist_3_joint: 0.502